lightning/src/ln/functional_tests.rs

```diff
@@ -7586,33 +7586,7 @@ fn test_priv_forwarding_rejection() {
 
 	let chan_id_1 = create_announced_chan_between_nodes_with_value(&nodes, 0, 1, 1_000_000, 500_000_000, InitFeatures::known(), InitFeatures::known()).2;
 
-	// Note that the create_*_chan functions in utils requires announcement_signatures, which we do
```

Comment:
Looks like you somehow ended up with some upstream changes here - can you rebase on upstream/main?

Reply:
That specific line is actually intentionally changed by me, to use the new utility function I created in `lightning/src/ln/functional_test_utils.rs` in the functional_tests as well. Hence my comment above if it's suitable to include that commit e29c401 in this PR or if I should drop it :)

But I'll make sure to fetch the latest upstream and rebase this on that!